### ISSUE TITLE
ModuleNotFoundError: No module named 'urllib3'

### ISSUE THREAD

**Issue by rgilton** [cobrateam/splinter#1068]
New to using splinter, just pip installed into a venv, and get a `ModuleNotFoundError` when trying to import:

```
% pip install splinter
Looking in indexes: https://pypi.org/simple
Collecting splinter
  Using cached splinter-0.18.1-py3-none-any.whl (37 kB)
Installing collected packages: splinter
Successfully installed splinter-0.18.1

% python -c "from splinter import Browser"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/rob/tmp/v/lib64/python3.10/site-packages/splinter/__init__.py", line 5, in <module>
    from splinter.browser import Browser  # NOQA
  File "/home/rob/tmp/v/lib64/python3.10/site-packages/splinter/browser.py", line 12, in <module>
    from urllib3.exceptions import MaxRetryError
ModuleNotFoundError: No module named 'urllib3'
```

**Comment by jsfehler** [comment]
@rgilton Are you using splinter without selenium?

**Comment by jsfehler** [comment]
@rgilton I asked the question because the answer determines if the bug is a simple fix or not.

**Comment by jsfehler** [comment]
Fixed by #1123
